src/huggingface_hub/repository.py

```diff
@@ -365,6 +365,7 @@ def __init__(
         git_email: Optional[str] = None,
         revision: Optional[str] = None,
         private: bool = False,
+        skip_lfs: bool = False,
```

Comment:
The `skip_lfs` argument isn't super clear, what about `download_lfs_fles` set to `True` by default?

Reply:
I've renamed it to `skip_lfs_files`, if that's ok for you

Reply:
Your call, but I also personally think that `skip_lfs` (or `skip_lfs_files`) defaulting to False is clearer (and consistent w/ the env var)

Reply:
Yes, sounds good